### DISCUSSION TITLE
小狼毫在Win10上会卡死，这个有解决的方法么？

### DISCUSSION THREAD

**Discussion by YummyCocoa** [Mintimate/oh-my-rime#3] [Q&A] [answered]
在小狼毫上，不管是否使用该配置，在频繁打字后，都有可能卡死…… 大家有没有解决的方法？

目前使用的是[小狼毫 0.14.3](https://github.com/rime/weasel/releases/tag/0.14.3)

**Comment by Mintimate** [comment, marked as answer]
我平时主要使用macOS上的鼠须管，刚刚在Windows上测试了一下，确实可能会有闪退的情况，而且根据反馈，在Win7上最为明显。目前的解决方法，建议使用[fxliang](https://github.com/fxliang)大佬维护的新版本小狼毫：
<img width="1392" alt="image" src="https://user-images.githubusercontent.com/43734212/235292038-2fc72763-91d3-4790-b79f-3d6389ca7f3a.png">

不清楚后续fxliang大佬维护的小狼毫是否会合并到主分支，但是确实比原版好。

还有就是五笔98版本的rime，也比原版好：
- [五笔98: http://www.98wubi.com/](http://www.98wubi.com/)

**Comment by yanhuacuo** [comment]
结构稍作调整，即可使用【小狼毫助手】的【进程守护】功能：

https://github.com/Mintimate/oh-my-rime/discussions/19

![QQ截图20231221232206](https://github.com/Mintimate/oh-my-rime/assets/3107994/807afeb8-8d84-46c1-959f-18cfd0334179)
